### PR TITLE
New OpenStack Cheat Sheet

### PR DESCRIPTION
**Pull request by mohan08p**
What does your Instant Answer do?

Show all The OpenStack basic commands as cheat sheet.

What is the data source for your Instant Answer? (Provide a link if possible)

http://docs.openstack.org/user-guide/cli_cheat_sheet.html

Why did you choose this data source?

OpenStack is a cloud operating system that manages your complete infrastructure through data-center. So, it is very helpful and easy for them to apply the commands.

Are there any other alternative (better) data sources?

No

Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)

DevOPs, SystemAdmins, Open source, Git users, Geeks, etc.

Is this Instant Answer connected to a DuckDuckHack Instant Answer idea?

Yes, https://duck.co/ideas/idea/6345/openstack-cheat-sheet

Are you having any problems? Do you need our help with anything?

No

What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)
openstack_snapshot

---

IA Page: https://duck.co/ia/view/openstack_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
@mohan08p Hey!
        
Thanks for taking the time to contribute! We really appreciate it.

We work closely with every contributor to make Instant Answers the best they can be, so we appreciate your patience as we look over your code. From here, the process usually goes like this:

1. Pull Request is reviewed by the DuckDuckGo staff and community
2. Staff and community will leave feedback with any necessary updates to the function or design.
3. Once you've made any necessary corrections, then your Instant Answer will be merged and deployed live on DuckDuckGo!

If you have any questions along the way, feel free to ask them here. Our staff and community are also available on Slack to answer any questions you may have. If you'd like to join us there please send an email to <quackslack@duckduckgo.com>.

>More Info: https://duck.co/duckduckhack/submission_and_review

Thanks!

**Comment by jagtalon** [comment]
@mohan08p checking it out!

**Comment by jagtalon** [comment]
@mohan08p add an alias (https://github.com/duckduckgo/zeroclickinfo-goodies/blob/master/share/goodie/cheat_sheets/json/age-of-mythology.json#L9) for "open stack" so that it triggers with "open stack cheat sheets".

**Comment by jagtalon** [comment]
@mohan08p It's a bit packed with information and it's pretty hard to read. Is it possible to make the descriptions / examples more succinct?

screen shot 2015-10-15 at 4 44 05 pm

**Comment by mohan08p** [comment]
@jagtalon thanks. please review the changes. :)

File: share/goodie/cheat_sheets/json/openstack.json [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    ],
+    "section_order": [
+	"Identity (keystone)",
+        "Images (glance)",
+        "Compute (nova)",
+        "Networking (neutron)",
+        "Block Storage (cinder)",
+        "Volume Management inside Instance",
+	"Object Storage (swift)"
+    ],
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/openstack.json]: Seeing as this is a `terminal` program, let's use the Terminal template: `"template_type": "terminal",

**Comment by moollaza** [comment]
@mohan08p this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by mohan08p** [comment]
@jagtalon , @moollaza  thank you so much sir. :) I am glad that I learned many things from you.

**Comment by moollaza** [comment]
@mohan08p congrats! It's live: https://duckduckgo.com/?q=OpenStack+Cheat+Sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Pull request merged**